contracts/Vault.vy

```diff
@@ -1142,6 +1142,15 @@ def _expectedReturn(_strategy: address) -> uint256:
         return 0  # Covers the scenario when block.number == strategy_activation
 
 
+@view
+@external
+def availableDepositLimit() -> uint256:
+    if self.depositLimit >= self._totalAssets():
```

Comment:
I know I said otherwise in the issue, but if they're equal, we can save some gas by returning 0
```suggestion
    if self.depositLimit > self._totalAssets():
```